staking/tests/utils/api_utils.ts

```diff
@@ -289,7 +289,7 @@ class MockProposalCreator {
       GovernanceAccountType.ProposalV2
     );
 
-    const serializedProp = serialize(schema, proposal, GovernanceBinaryWriter);
+    const serializedProp = serialize(schema, proposal);
```

Comment:
not sure what this does tho.

Reply:
It was failing a type check so I just fixed it